test/e2e_node/services/kubelet.go

```diff
@@ -194,6 +201,58 @@ func createPodManifestDirectory() (string, error) {
 	return path, nil
 }
 
+// createKubeconfig creates a kubeconfig file at the fully qualified `path`.
+func createKubeconfig(path string) error {
+	kubeconfig := `apiVersion: v1
+kind: Config
+users:
+- name: kubelet
+clusters:
+- cluster:
+    server: ` + getAPIServerClientURL() + `
+    insecure-skip-tls-verify: true
+  name: local
+contexts:
+- context:
+    cluster: local
+    user: kubelet
+  name: local-context
+current-context: local-context`
+
+	f, err := os.Create(path)
+	defer f.Close()
+	if err != nil {
+		return err
+	}
+	_, err = f.WriteString(kubeconfig)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func kubeconfigCWDPath() (string, error) {
+	cwd, err := os.Getwd()
```

Comment:
CWD for e2e_node services runner should be under the time-stamped dir that corresponds to an individual test run, which should always be writable because it's under `/tmp`.

 Multiple Kubelets should never be run in parallel on the same node, even if multiple tests are being run in parallel.

Why would dirty git trees be an issue here?